test/inspector/test-inspector-port-zero-cluster.js

```diff
@@ -6,27 +6,29 @@ const assert = require('assert');
 const cluster = require('cluster');
 
 if (cluster.isMaster) {
+  cluster.on('online', common.mustCall((worker) => worker.send('debugPort'), 3));
+  cluster.on('error', (err) => assert.fail(err));
   const ports = [];
-  for (const worker of [cluster.fork(),
-                        cluster.fork(),
-                        cluster.fork()]) {
-    worker.on('message', common.mustCall((message) => {
-      ports.push(message.debugPort);
-      worker.kill();
-    }));
-    worker.send('debugPort');
+  function serialFork() {
+    return new Promise((res) => {
+      const worker = cluster.fork();
+      worker.on('message', common.mustCall((message) => {
+        worker.kill();
+        ports.push(message.debugPort);
+        res();
+      }));
+    });
   }
-  process.on('exit', () => {
-    ports.sort();
+  serialFork().then(serialFork).then(serialFork).then(common.mustCall(() => {
     assert.strictEqual(ports.length, 3);
     assert(ports.every((port) => port > 0));
     assert(ports.every((port) => port < 65536));
     assert.strictEqual(ports[0] + 1, ports[1]);  // Ports should be consecutive.
```

Comment:
While we're in here fixing up the test, I wonder if we want to change this? It's not really guaranteed that the ports are sequential, is it? If nothing else, won't it wrap around to lower port numbers after 65535? I wonder if we want to change this to a check that we have three unique port numbers rather than sequential port numbers?